### ISSUE TITLE
double spending

### ISSUE THREAD

**Issue by novidi** [iron-fish/ironfish#693]
hi .
![Untitled](https://user-images.githubusercontent.com/93491527/145616121-e584f8c6-8dd1-4669-8920-32c312840d68.jpg)
 i just mined one block but dosnt show on my ironfish profile an also show this messege in docker:
Successfully mined block 000000000339037aab49a18fb4c0d755de387680e327351aff6415e660a19018 (16228) has 2 transactions
my grafitti: noweedi

**Comment by adasari** [comment]
Same issue 

```
root@almuezza:/home/almuezza# journalctl -u ironfishd | grep mined
Dec 08 14:30:17 almuezza yarn[920594]: Successfully mined block 0000000001e887087566270f75e218e02e257a1d687e00abf784f17c4e02dee2 (13790) has 2 transactions
Dec 08 14:30:19 almuezza yarn[920594]: Failed to add mined block to chain with reason Invalid spend
Dec 08 18:32:36 almuezza yarn[31445]: - /root/ironfish/ironfish-cli/build/commands/miners/mined.js
Dec 10 04:00:45 almuezza yarn[1273164]: Successfully mined block 0000000005a9ea9033719822552d632ab1a983fe2593a82f4c1a4b01a81a4c5c (15877) has 2 transactions
Dec 10 12:37:16 almuezza yarn[1316339]: Successfully mined block 0000000005b3a66052c29dd20f54db7bb0ddc0c9eb9d04684e3086bd08051f1e (16120) has 4 transactions
Dec 10 12:37:21 almuezza yarn[1316339]: Failed to add mined block to chain with reason Double spend
Dec 10 14:33:59 almuezza yarn[1472539]: Successfully mined block 000000000bc92fee60ac7352f72f55231b066bf8c29918cabd5e1922de028e66 (16243) has 1 transactions
Dec 10 17:29:55 almuezza yarn[1472539]: Successfully mined block 000000000b9d6caa2fc4a943c95415b0cc3a9cc83c1c1f411e4cea33873c4e85 (16343) has 2 transactions
Dec 10 17:29:56 almuezza yarn[1472539]: Failed to add mined block to chain with reason Invalid spend
Dec 11 02:32:00 almuezza yarn[1700384]: Successfully mined block 000000001426013ac94bfa1663aae3524ec24c55be2541673aced123d1f57da4 (16583) has 4 transactions
Dec 11 02:32:03 almuezza yarn[1700384]: Failed to add mined block to chain with reason Double spend
```

**Comment by deekerno** [comment]
We put out fixes for this issue with the release of v0.1.12. If you are still encountering this issue, feel free to re-open this issue.
